### PR TITLE
Add Prisma

### PR DESCRIPTION
**Pull request by catalinmiron**
Prisma - Database tools for modern application development

**[https://github.com/catalinmiron/awesome-prisma#readme](https://github.com/catalinmiron/awesome-prisma)**

Prisma is a performant open-source GraphQL ORM-like layer doing the heavy lifting in your GraphQL server. It turns your database into a GraphQL API which can be consumed by your resolvers via GraphQL bindings.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
    - #1498
    - #1514 

- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by victorshinya** [comment]
Thanks for contributing @catalinmiron 

Take a quick look:
- I would suggest you add a LICENSE file to officially make a CC0 repository. Take a look at this repository as an example (https://github.com/victorshinya/awesome-ibmcloud/blob/master/LICENSE).

**Comment by catalinmiron** [comment]
@victorshinya thanks for commenting out. It already have a license: https://github.com/catalinmiron/awesome-prisma#license. This repository was created using the yeoman plugin so we're good to go and I'm using an SVG badge for CC0.

File: readme.md [hunk 1]
```diff
@@ -601,6 +601,7 @@
 - [Humane Technology](https://github.com/engagingspaces/awesome-humane-tech#readme) - Open source projects that help improve society.
 - [Speakers](https://github.com/karlhorky/awesome-speakers#readme) - Conference and meetup speakers in the programming and design community.
 - [Software Patreons](https://github.com/uraimo/awesome-software-patreons#readme) - Fund individual programmers or the development of open source projects.
+- [Prisma](https://github.com/catalinmiron/awesome-prisma#readme) - Database tools for modern application development
```
Comment (sindresorhus) [inline readme.md]: The descriptions needs to be more informative and less of a tagline. You also forgot the dot at the end.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/catalinmiron/awesome-prisma
✖ Linting

  readme.md:7:85
  ✖    7:85   Titles should use ' as a quote                                                                           link-title-style
  ✖   33:1    ToC item ":family_man_woman_girl_boy: Community" link "#family_man_woman_girl_boy--community" not found  awesome/toc
  ✖   37:59   Titles should use ' as a quote                                                                           link-title-style
  ✖   38:82   Titles should use ' as a quote                                                                           link-title-style
  ✖   39:73   Titles should use ' as a quote                                                                           link-title-style
  ✖   40:74   Titles should use ' as a quote                                                                           link-title-style
  ✖   41:114  Titles should use ' as a quote                                                                           link-title-style
  ✖   42:57   Titles should use ' as a quote                                                                           link-title-style
  ✖   45:61   Titles should use ' as a quote                                                                           link-title-style
  ✖   47:106  Titles should use ' as a quote                                                                           link-title-style
  ✖   48:94   Titles should use ' as a quote                                                                           link-title-style
  ✖   49:98   Titles should use ' as a quote                                                                           link-title-style
  ✖   50:115  Titles should use ' as a quote                                                                           link-title-style
  ✖   57:67   Titles should use ' as a quote                                                                           link-title-style
  ✖   58:12   Link to https://www.prisma.io/tutorials/build-a-graphql-server-with-prisma-ct02/ is dead                 no-dead-urls
  ✖   64:12   Link to https://www.prisma.io/tutorials/archived/deploy-prisma-to-zeit-now-ct16/ is dead                 no-dead-urls
  ✖   65:105  Titles should use ' as a quote                                                                           link-title-style
  ✖   66:135  Titles should use ' as a quote                                                                           link-title-style
  ✖   84:10   Link to https://github.com/graphcool/prisma/tree/master/examples/auth is dead                            no-dead-urls
  ✖   85:10   Link to https://github.com/graphcool/prisma/tree/master/examples/github-auth is dead                     no-dead-urls
  ✖   85:11   Text "Github" should be written as "GitHub"                                                              awesome/spell-check
  ✖   86:13   Text "Github" should be written as "GitHub"                                                              awesome/spell-check
  ✖   97:10   Link to https://github.com/graphcool/prisma/tree/master/examples/subscriptions is dead                   no-dead-urls
  ✖  101:10   Link to https://github.com/graphcool/prisma/tree/master/examples/file-handling-s3 is dead                no-dead-urls
  ✖  123:152  Titles should use ' as a quote                                                                           link-title-style
  ✖  124:166  Titles should use ' as a quote                                                                           link-title-style

  26 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
> Includes a project logo/illustration whenever possible.
**The image should link to the project website or any relevant website.**
**The image should be high-DPI. Set it to maximum half the width of the original image.**

---

The heading should be `Awesome Prisma`, not `awesome-prisma`.

---

`- hands-on resources` => `- Hands-on resources`

`- Video Tutorial` => `- Video tutorial`

---

https://github.com/catalinmiron/awesome-prisma/blob/master/contributing.md should be customized for your list.

It's very clear it's just a copy-paste because of `Use the following format: [List Name](link)`. (Note `List`).

File: readme.md (hunk 1 above)
Comment (catalinmiron) [reply readme.md]: Done. Please take another look. Thanks!

**Comment by catalinmiron** [comment]
@sindresorhus please take another look. Thanks!

**Comment by jthegedus** [comment]
This passes the `awesome-lint` tests now.

As previously mentioned, GitHub doesn't recognize the license as it is merely linking to a definition of the CCO license and not actually contained within the repo. I'm not sure if this is a pass or fail as far as the PR guidelines go.

**Comment by catalinmiron** [comment]
@jthegedus @victorshinya I've added the `LICENSE` file. Please take another look.

**Comment by jthegedus** [comment]
Can confirm that GitHub recognizes this as a CC0 repo now

**Comment by sindresorhus** [comment]
> The image should be high-DPI. Set it to maximum half the width of the original image.

Not fixed.

**Comment by sindresorhus** [comment]
@catalinmiron Ping

**Comment by sindresorhus** [comment]
You also need to fix the merge conflict.

**Comment by sindresorhus** [comment]
`Prisma` in `Prisma is a performant open-source GraphQL ORM-like layer doing the heavy lifting in your GraphQL server.` should be linkified to its website.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/catalinmiron/awesome-prisma
✖ Linting

  readme.md:3:1
  ✖   1:1    Missing Awesome badge after the main heading                                                                                                       remark-lint:awesome/badge
  ✖   1:145  Remove empty section: "<p align="center"><img src="https://raw.githubusercontent.com/catalinmiron/awesome-prisma/master/awesome-prisma.png"></p>"  remark-lint:no-empty-sections
  ✖   3:1    Don’t use multiple top level headings (1:1)                                                                                                        remark-lint:no-multiple-toplevel-headings
  ✖  15:1    Table of Contents must be the first section                                                                                                        remark-lint:awesome/toc
  ✖  43:10   Link to https://github.com/coformatique/prisma-auth0-angular-starter is dead                                                                       remark-lint:no-dead-urls
  ✖  84:10   Link to https://github.com/coformatique/prisma-auth0-starter is dead                                                                               remark-lint:no-dead-urls
  ✖  97:15   Link to https://manticarodrigo.com/file-handling-s3-prisma-graphql-yoga/ is dead                                                                   remark-lint:no-dead-urls

  7 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by catalinmiron** [comment]
I'll fix the issues and ping you, I was on vacation and didn't had access to my laptop. Thanks!

**Comment by sindresorhus** [comment]
Ping

**Comment by catalinmiron** [comment]
@sindresorhus sorry for my late response, can you please take another look? Thanks!

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1126392334940131334

**Pull request merged**